src/pages/FAQ.tsx

```diff
@@ -2,105 +2,109 @@ import { PageContainer } from '@ant-design/pro-layout';
 import { Collapse, Divider } from 'antd';
 import React from 'react';
 import { useIntl } from 'umi';
-
 const { Panel } = Collapse;
 
 export default (): React.ReactNode => {
   const intl = useIntl();
 
-  document.title = 'FAQ - Metal Gear Online';
-
+  document.title = 'FAQ - Metal Gear Online 2 Fan Revival';
   return (
     <PageContainer>
       <Collapse defaultActiveKey={['1']}>
-        <Panel header={intl.formatMessage({ id: 'app.faq.sysreqtitle' })} key="1">
+        <Panel header="Registration Process" key="1">
           <div>
-            <p>{intl.formatMessage({ id: 'app.faq.sysreq' })}</p>
-            <Divider />
-            <p>
-              <b>{intl.formatMessage({ id: 'app.minimum' })}</b>
-            </p>
-            <p>
-              <i>These CPUs are the minimum recommended to maintain 30 FPS in 12 player lobbies. </i>
-            </p>
-            <p>
-              <i>It is possible to play on weaker hardware than this but we cannot recommend it.</i>
-            </p>
-            <ul>
-              <li>CPU: Intel i5 10400 or AMD Ryzen 3600</li>
-              <li>GPU: Anything with Vulkan Support</li>
-            </ul>
+            <p>To register, please visit <a href="https://mgo2pc.com/register">our registration page</a>. We advise using a Gmail account for registration as Hotmail/Outlook emails have been experiencing issues lately. Your username should be at least 8 characters long, containing only letters and numbers, without spaces or special characters. This username will also serve as your Game ID, and the chosen password will be your game password. Once your account is created, it is not necessary to change your password, even if redirected to a password change page.</p>
+          </div>
+        </Panel>
+        <Panel header="Email Confirmation Issues" key="2">
+          <div>
+          <p>If you do not receive an email confirmation, it is possible that a Hotmail/Outlook email address was used, which might cause delays. However, account activation is not required. Please ensure you remember your username and password. If forgotten, assistance can be sought by opening a ticket in our <a href="https://discord.com/invite/MNqmzuW2wH">Discord channel</a>.</p>
+          </div>
```

Comment:
These are the only ones that I'm not a fan of. They're too wordy, and sound a bit awkward. I don't think we need the Registration Process entry at all.